internal/cmd/trust/addrootkey/addrootkey.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package addrootkey
+
+import (
+	"os"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/trust/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p          *persistent.Options
+	newRootKey string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.newRootKey,
+		"root-key",
+		"",
+		"root key to add to root of trust",
+	)
+	cmd.MarkFlagRequired("root-key") //nolint:errcheck
+}
+
+func (o *options) Run(cmd *cobra.Command, _ []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+
+	rootKeyBytes, err := os.ReadFile(o.p.SigningKey)
+	if err != nil {
+		return err
+	}
+
+	newRootKeyBytes, err := common.ReadKeyBytes(o.newRootKey)
+	if err != nil {
+		return err
+	}
+
+	return repo.AddRootKey(cmd.Context(), rootKeyBytes, newRootKeyBytes, true)
+}
+
+func New(persistent *persistent.Options) *cobra.Command {
+	o := &options{p: persistent}
+	cmd := &cobra.Command{
+		Use:     "add-root-key",
+		Short:   "Add Root key to gittuf root of trust",
+		Long:    `This command allows users to add a new trusted key to the root. Note that authorized keys can be specified from disk using the custom securesystemslib format, from the GPG keyring using the "gpg:<fingerprint>" format, or as a Sigstore identity as "fulcio:<identity>::<issuer>".`,
```

Comment:
This is currently incorrect, something we ought to fix in addpolicykey too until #229 is fixed. Or at the very least, these can be specified and it'll work but signing the metadata will fail.

maybe we can leave it in and focus on that through #229 instead.

Reply:
Should I just take out the last sentence of the long --help for add root key?

Reply:
for now, yes.

Reply:
Also bump on this one.